pkg/spec/createconfig.go

```diff
@@ -433,6 +446,9 @@ func (c *CreateConfig) GetContainerCreateOptions() ([]libpod.CtrCreateOption, er
 	if c.CgroupParent != "" {
 		options = append(options, libpod.WithCgroupParent(c.CgroupParent))
 	}
+	if c.Detach {
+		options = append(options, libpod.WithExitCommand(createExitCommand(runtime)))
```

Comment:
We should probably do this unconditionally, and remove the calls to `Cleanup()` in `podman run`

Reply:
I would prefer to wait for this, make sure this is working well before we make it the default.